### PR TITLE
GH-110: Build of N4JS type definition projects still using an obsolete nexus server

### PR DESCRIPTION
**Pull request by mor-n4**
See #110.

### CODE REVIEW

**Comment by mor-n4** [comment]
See build:
http://n4ide1-jenkins.service.cd-dev.consul/view/IDE/job/N4JSD-Publish-Nightly_TEMP/1/consoleFull

File: publish.sh [hunk 1]
```diff
@@ -31,7 +31,7 @@ export PUBLISH_SINGLE_NPM="${DIR}/publishSingleNPM.sh"
 
 
 echo "== Start publish script"
-echo "dir of scipt is $DIR"
+echo "dir of script is $DIR"
```
Comment (mmews-n4) [inline publish.sh:34]: 👍

**Review by mmews-n4 — APPROVED** [review]
👍 Thanks also for updating the Dockerfile

**Comment by mor-n4** [comment]
Thanks, merging!

**Pull request merged**